romancal/tweakreg/tests/test_astrometric_utils.py

```diff
@@ -696,26 +701,19 @@ def test_get_catalog_using_epoch(ra, dec, epoch):
     assert mad_std(returned_dec - expected_dec) < 2.8e-9
 
 
-def test_get_catalog_timeout():
+def test_get_catalog_timeout(monkeypatch):
     """Test that get_catalog can timeout."""
 
-    with pytest.raises(Exception) as exec_info:
-        for dt in np.arange(1, 0, -0.01):
-            try:
-                get_catalog(10, 10, sr=0.1, catalog="GAIADR3", timeout=dt)
-            except requests.exceptions.ConnectionError:
-                # ignore if it's a connection error instead of timeout
-                pass
+    monkeypatch.setattr("requests.get", MockTimeoutError)
 
-    assert exec_info.type == requests.exceptions.Timeout
```

Comment:
I make this test simplification simply because `get_catalog(10, 10, sr=0.1, catalog="GAIADR3")` can be called by earlier tests meaning the cache will actually have the value and no connection will happen. We could simulate a real timeout if we make sure this test is run before any other tests or choose some set of inputs that are valid query inputs because you get a connection error if one makes a query with inputs that the server doesn't know how to handle because this is essentially just looking at a URL.